assets/js/romo/base.js

```diff
@@ -601,10 +601,11 @@ Romo.prototype.on = function(elem, eventName, fn) {
   proxyFn._romofid = this._fn(fn)._romofid;
 
   var key = this._handlerKey(elem, eventName, proxyFn);
-  if (!this._handlers[key]) {
-    elem.addEventListener(eventName, proxyFn);
-    this._handlers[key] = proxyFn;
+  if (this._handlers[key] === undefined) {
+    this._handlers[key] = [];
```

Comment:
@jcredding can we jux this logic with the `.push` statement below (move it after the `addEventListener` statement please?